tutor/templates/course.html

```diff
@@ -14,7 +14,7 @@ <h5 class="card-title">{{resource.title}}</h5>
                         <p class="card-text">
                             {{resource.content}}
                         </p>
-                        <a href="{{resource.link}}" class="card-link">Click here</a>
+                        <a href="{{resource.link}}" class="card-link">{{resource.link}}</a>
```

Comment:
I wonder if you really wand to show the URL as-is or perhaps make the title be the link, or some icon.

Reply:
@ifireball Commited changes according to your comments, is it ok to merge?

Reply:
@Michal-Shamir I thought I already set my approval... my comments were just suggestions. In any case since you've now moved the PR the draft state I'm guessing you don't want to merge at this point?